### ISSUE TITLE
online.boscaceoil.net DNS not resolving

### ISSUE THREAD

**Issue by nornagon** [dy-dx/boscaceoil-online#7]
```
$ host online.boscaceoil.net
Host online.boscaceoil.net not found: 3(NXDOMAIN)
```

Would love to see this come back online :)

**Comment by apes0** [comment]
yeah, it was really nice, especially since adobe air has ended support so its pretty hard to get bosca ceoil on certain linux distros, for some even impossible without wine!

**Comment by UCYT5040** [comment]
Can confirm it is having issues, I'm gonna try hosting it myself.
